### PR TITLE
Actualització ajuda 0-16 (AE-230 i AE-230-1)

### PR DESCRIPTION
**Pull request by mdsilberberg**

### CODE REVIEW

File: src/family/FamilyForm.js [hunk 1]
```diff
@@ -62,6 +63,7 @@ const FamilyForm = (props: Props) => {
     sustentadorsSolitarisAmbPossiblesParelles,
     t
   } = props;
+  const year = moment().subtract(1, 'year').format('YYYY');
```
Comment (jvalduvieco) [inline src/family/FamilyForm.js]:
```suggestion
  const previous_year = moment().subtract(1, 'year').format('YYYY');
```

File: public/locales/es/translations.json [hunk 2]
```diff
@@ -171,7 +171,7 @@
     "una_persona_que_no_conviu": "Una persona que no convive",
     "ningu_mes": "Nadie más",
     "te_carnet_monoparental": "¿Tiene el carnet de familia monoparental?",
-    "familia_usuaria_css": "¿Esta familia es usuaria de servicios sociales en seguimiento en un centro de Servicios Sociales o servicio especializado del Ayuntamiento de Barcelona desde antes del 31/12/2017?",
+    "familia_usuaria_css": "¿Esta familia es usuaria de servicios sociales en seguimiento en un centro de Servicios Sociales o servicio especializado del Ayuntamiento de Barcelona desde antes del 31/12/{year}?",
```
Comment (jvalduvieco) [inline public/locales/es/translations.json]:
```suggestion
    "familia_usuaria_css": "¿Esta familia es usuaria de servicios sociales en seguimiento en un centro de Servicios Sociales o servicio especializado del Ayuntamiento de Barcelona desde antes del 31/12/{previous_year}?",
```

File: src/family/FamilyForm.js [hunk 3]
```diff
@@ -161,9 +163,11 @@ const FamilyForm = (props: Props) => {
                         </YesNoQuestion>}
 
                         <YesNoQuestion name={'usuari_serveis_socials.' + familia.ID} validate={[required]}>
-                          <Trans i18nKey='familia_usuaria_css'>Aquesta família és usuària de serveis socials en
-                            seguiment a un CSS o servei
-                            especialitzat de l'Ajuntament de Barcelona des d'abans del 31/12/2017?</Trans>
+                          <Trans
+                            i18nKey='familia_usuaria_css'
+                            year={year} >
```
Comment (jvalduvieco) [inline src/family/FamilyForm.js]:
```suggestion
                            previous_year={previous_year} >
```

File: src/family/FamilyForm.js [hunk 4]
```diff
@@ -161,9 +163,11 @@ const FamilyForm = (props: Props) => {
...
+                            Aquesta família és usuària de serveis socials en seguiment a un CSS o servei especialitzat de l'Ajuntament de Barcelona des d'abans del 31/12/{{ year }}?
```
Comment (jvalduvieco) [inline src/family/FamilyForm.js]:
```suggestion
                            Aquesta família és usuària de serveis socials en seguiment a un CSS o servei especialitzat de l'Ajuntament de Barcelona des d'abans del 31/12/{{ previous_year }}?
```

File: src/results/PersonalBenefits.js [hunk 5]
```diff
@@ -23,10 +23,10 @@ class PersonalBenefits extends React.Component<PersonalBenefitsProps> {
         name: <Trans i18nKey='AE_230_mensual_title'>Fons extraordinari d’ajuts d’emergència social per a infants de 0 a
           16 anys</Trans>,
         periode: <Trans i18nKey='mes'>mes</Trans>,
-        conditions: <Trans i18nKey='maxim_9_mesos'>màxim 9 mesos</Trans>,
+        conditions: <Trans i18nKey='maxim_8_mesos'>màxim 8 mesos</Trans>,
         url: this.props.t('link_AE_230'),
-        from: newDate(2018, 4, 25),
-        to: newDate(2018, 5, 25)
+        from: Moment().date(25).month(2).toDate(),
```
Comment (jvalduvieco) [inline src/results/PersonalBenefits.js]: This must be hardcoded as we don't have any information of 2020 dates. And should be consistent with AE_230_01

File: src/results/PersonalBenefits.js [hunk 6]
```diff
@@ -35,8 +35,8 @@ class PersonalBenefits extends React.Component<PersonalBenefitsProps> {
           monoparentals</Trans>,
         periode: <Trans i18nKey='any'>any</Trans>,
         url: this.props.t('link_AE_230'),
-        from: newDate(2018, 4, 25),
-        to: newDate(2018, 5, 25)
+        from: newDate(2019, 4, 25),
+        to: newDate(2019, 5, 25)
```
Comment (jvalduvieco) [inline src/results/PersonalBenefits.js]:
```suggestion
        to: newDate(2019, 5, 24)
```

**Comment by mdsilberberg** [comment]
@jvalduvieco changes have been implemented!

**Pull request merged**